src/Components/Aspire.Azure.AI.OpenAI/AzureOpenAISettings.cs

```diff
@@ -39,13 +42,56 @@ public sealed class AzureOpenAISettings : IConnectionStringSettings
     /// </remarks>
     public string? Key { get; set; }
 
+    /// <summary>
+    /// Gets or sets a boolean value that indicates whether the OpenTelemetry metrics are enabled or not.
+    /// </summary>
+    /// <remarks>
+    /// Azure AI OpenAI telemetry support is experimental, the shape of traces may change in the future without notice.
+    /// It can be enabled by setting "OpenAI.Experimental.EnableOpenTelemetry" <see cref="AppContext"/> switch to true.
+    /// Or by setting "OPENAI_EXPERIMENTAL_ENABLE_OPEN_TELEMETRY" environment variable to "true".
+    /// </remarks>
+    /// <value>  
+    /// The default value is <see langword="false"/>.  
```

Comment:
It's not necessarily `false` - it defaults to the value of the env var / appcontext switch. I think we could just leave this `value` section off until the env var goes away.

Reply:
applied